src/pages/amazon-sales-channel/index.md

```diff
@@ -1,6 +1,30 @@
 ---
-title: Amazon Sales Channel reference app overview
-description: 
+title: Amazon Sales Channel on App Builder overview
+description: Learn how you can use Amazon Sales Channel on App Builder as a reference app to build your own Adobe Commerce apps.
 ---
 
-# Amazon Sales Channel reference app overview
+# Amazon Sales Channel on App Builder overview
+
+Amazon Sales Channel on App Builder is a flagship reference app developed by Adobe Commerce to accelerate merchant and partner adoption of out-of-process extensibility. This app showcases how Adobe Developer App Builder can be used to create integrations and extensions without modifying the core Commerce application. Amazon Sales Channel on App Builder is focused on demonstrating how key capabilities and extensibility use cases can be accomplished using App Builder. Partners and developers can leverage the application code, best practice guides, and documentation to confidently create their own next-generation extensions and integrations.
```

Comment:
```suggestion
Amazon Sales Channel on App Builder is a flagship reference app developed by Adobe Commerce to accelerate merchant and partner adoption of out-of-process extensibility. This app showcases how Adobe Developer App Builder can create integrations and extensions without modifying the core Commerce application. Amazon Sales Channel on App Builder focuses on demonstrating how to accomplish key capabilities and extensibility use cases with App Builder. Partners and developers can leverage the application code, best practice guides, and documentation to confidently create their own next-generation extensions and integrations.
```

Reply:
@keharper I think "develop" might work better than "accomplish" in the restructured 3rd sentence. Otherwise edits look good.